frontend/src/app/components/FeaturedExperiment/index.scss

```diff
@@ -209,7 +209,7 @@
   }
 
   .main-install__available {
-    display: none;
+      font-size: 13px;
```

Comment:
This should be a multiple of [`$font-unit`](https://github.com/mozilla/testpilot/blob/master/frontend/src/styles/_utils.scss#L56)